src/pages/Home/Home.page.jsx

```diff
@@ -1,23 +1,65 @@
 import React, { useRef } from 'react';
-import { Link, useHistory } from 'react-router-dom';
+// import { Link, useHistory } from 'react-router-dom';
 
-import { useAuth } from '../../providers/Auth';
+// import { useAuth } from '../../providers/Auth';
+import styled from 'styled-components';
+import youtubeData from '../../youtube-videos-mock';
 import './Home.styles.css';
 
+const VideoCard = styled.div`
```

Comment:
The same comment as on App component

Reply:
Done. Exported the styled component to a separate file